data/domain/breeding.tsx

```diff
@@ -250,6 +282,11 @@ export class Pet {
                 text: randoList[91].data.elements[shinyIndex].replace(/_/g, " ")
             }, 0);
         });
+        allPets.forEach(pet => {
+            const sameWorldPets = allPets.filter(findPet => findPet.data.world == pet.data.world).slice().sort((pet1, pet2) => pet1.data.unlockOrder > pet2.data.unlockOrder ? 1 : -1);
+            pet.indexInWorld = sameWorldPets.findIndex(findPet => findPet.index == pet.index);
```

Comment:
I don't mind the current implementation but maybe something we can ask from the wiki in the future